pkg/sql/pg_catalog.go

```diff
@@ -2543,6 +2805,1174 @@ https://www.postgresql.org/docs/9.5/catalog-pg-shseclabel.html`,
 	},
 }
 
+var pgCatalogAmopOidIndexTable = virtualSchemaTable{
+	comment: "pg_amop_oid_index was created by pg_catalog_test and is currently unimplemented",
```

Comment:
nit: since this message is user-visible, maybe we should just say "was created for compatibility"